x/ccv/consumer/ibc_module.go

```diff
@@ -213,6 +213,7 @@ func (am AppModule) OnRecvPacket(
 		errAck := channeltypes.NewErrorAcknowledgement("cannot unmarshal CCV packet data")
 		ack = &errAck
 	} else {
+		// TODO: add possibility to receive ReplaceValidatorSet if it's the first time switching to consumer chain
```

Comment:
`ReplaceValidatorSet` should be a method in the staking module called by the consumer CCV module.